### PR TITLE
Feat : Home Page Added

### PR DESCRIPTION
**Pull request by purohitdheeraj**
This Branch contains the Home page of Ecommerce

### CODE REVIEW

File: components/home/home.css [hunk 1]
```diff
@@ -54,34 +69,38 @@
...
     gap: var(--TWO-REM);
     margin: 0 auto;
+    justify-content: center;
 }
 
-
-
 .type{
-    position: relative;
+    /* border: thin solid; */
```
Comment (DivyanshSareen) [inline components/home/home.css]: remove comment

**Review by DivyanshSareen — APPROVED** [review]
Code looks nice, great theme for the site. Good to merge

File: components/home/home.css [hunk 2]
```diff
@@ -7,30 +7,42 @@
...
     width: 80%;
 }
 
-.btn{
-    
-    padding: 4px;
-}
 .navbar a{
     font-size: 1.6rem;
 }
```
Comment (developer-anand) [inline components/home/home.css:21]: remove extra spaces

**Review by developer-anand — APPROVED** [review]
Good to merge.

File: main.css [hunk 3]
```diff
@@ -17,9 +17,6 @@
     box-shadow: var(--BOX-SHADOW);
 }
 
-.container > *{
-    /* margin-block: 1rem; */
-}
 
 /* this will make main always in center */
 .container > main{
```
Comment (deekshithmd) [inline main.css:22]: refactored code looks clean and good to merge

**Review by deekshithmd — APPROVED** [review]

**Pull request merged**